### ISSUE TITLE
Jupiter and other Solar System objects besides the Sun and Moon appear to not have any Events

### ISSUE THREAD

**Issue by cecomp64** [AstronomyAPI/Samples#122]
## Description of the issue

Thank you so much for this wonderful tool!  It would be nice if the Events API could include things like: rise, transit, set for every Solar System object.  It would also be nice if it included things like the Great Red Spot transit times and Gallilean Moon eclipses and transits for Juptier, and phases for Venus.

## What did you try and what didn't work

If I select any body other than Sun or Moon, I get an error response:

```js
data: { message: 'Invalid body' }
```

## Example code (if applicable)

```js
          pathname: '/api/v2/bodies/events/venus',
          search: '?latitude=37.25626720108698&longitude=-121.94236276119172&elevation=3&from_date=2024-12-17&to_date=2025-01-17&time=23:50:06&output=rows'
```

## Application ID (this will help trace the logs to see what went wrong)

`9d1d1b68-7400-4be9-99ba-c33941938bdb`

**Comment by astroanu** [comment]
Yes. Currently, it's only supported for bodies `sun` and `moon`.
